py3status/modules/net_rate.py

```diff
@@ -50,39 +65,36 @@ class Py3status:
     devfile = '/proc/net/dev'
     format = "{interface}: {total}"
     format_no_connection = ''
+    format_value = "{value:5.1f} {unit:>5s}"
```

Comment:
I cannot see any reason why we should give precision the light of day.

> The user shouldn't leave precision if they set format_value. By giving precedence to precision we force its removal if they want to use format_value.

I think if they set format_value precision should be ignored as they are setting a new param and just have not removed an obsolete one.